package.json

```diff
@@ -3,22 +3,20 @@
   "version": "2.0.5",
   "description": "",
   "scripts": {
-    "test": "mocha"
+    "test": "mocha --exit"
   },
   "dependencies": {
-    "debug": "2.6.9",
-    "mongodb-uri": "^0.9.7",
+    "debug": "4.1.1",
     "msgpack-js": "0.3.0",
-    "mubsub": "^1.4.0",
-    "socket.io-adapter": "^1.1.1",
+    "@nodebb/mubsub": "^1.6.0",
+    "socket.io-adapter": "^1.1.2",
```

Comment:
What if we dont bump up the version of socket io adapter ? Will it cause issues with nodebb/mubsub ?